app/config.py

```diff
@@ -16,11 +17,18 @@ class Settings(BaseSettings):
     db_port: str
     db_password: str
     db_user: str
-    secret_key: str
+
+    refresh_token_name: str = "refresh_token"
+    access_token_name: str = "access_token"
+    verify_token_secret_key: str
+    access_token_secret_key: str
+    refresh_token_secret_key: str
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Hard-coded secrets should be avoided in configuration files.

Consider using environment variables or secure vaults for storing secrets.